Fix links in IPFS desktop page

ElPaisano opened the pull request:
While going through https://docs.ipfs.tech/install/ipfs-desktop/, I noticed two busted links:
1) The link in the last sentence of the Windows install _The IPFS Desktop application has finished installing. You can now start to [add your site](https://docs.ipfs.tech/install/how-to/websites-on-ipfs/single-page-website/#add-your-site)._ returns a 404
2) The link in the last sentence of the Mac install _The IPFS Desktop application has finished installing. You can now start to [add your site](https://docs.ipfs.tech/install/ipfs-desktop/#add-your-site)._ doesn't go anywhere.

I'm guessing they were supposed to go to https://docs.ipfs.tech/how-to/websites-on-ipfs/single-page-website/ so I updated them accordingly. LMK if this is not the case.

TMoMoreau commented:
Weird, the page still exists, not sure why it's returning a 404.
https://docs.ipfs.tech/how-to/websites-on-ipfs/single-page-website/#add-your-site

We should try and keep the same link, the first half of the `/single-page-website/` page is just installation instructions anyway, which is where people who click the link are coming from.

Really not sure why it's 404-ing though. Maybe try and just paste the link above into the doc first, we know that one works.

ElPaisano commented:
@TMoMoreau thanks for looking. I made updates in https://github.com/ipfs/ipfs-docs/pull/1292/commits/6b0a31caf28a5850290c594eb0aa9e2a710550bf

I changed the links back to https://docs.ipfs.tech/how-to/websites-on-ipfs/single-page-website/#add-your-site per your comment above.

I think the 404 was being caused because a `.` was missing in the path, so the link was rendering as https://docs.ipfs.tech/install/how-to/websites-on-ipfs/single-page-website/#add-your-site when it should render as https://docs.ipfs.tech/how-to/websites-on-ipfs/single-page-website/#add-your-site. I added the missing `.`.

As for the other one, I think it just needed the full path, so I added that.

Both updated link seem to work.

TMoMoreau commented on docs/install/ipfs-desktop.md:
```diff
@@ -62,7 +62,7 @@ Or, if you'd rather use a package manager, check this [list of third-party packa
    ![The IPFS Desktop status bar menu in the Windows status bar.](./images/ipfs-desktop/install-windows-ipfs-desktop-status-bar.png)
 
-The IPFS Desktop application has finished installing. Now, learn how to [host a website using IPFS →](../how-to/websites-on-ipfs/single-page-website.md).
+The IPFS Desktop application has finished installing. Now, [add your site](../how-to/websites-on-ipfs/single-page-website.md#add-your-site).
```
Looks like there should still be a `/` between `/single-page-website.md` and `#add-your-site`?

Same thing for the other link that was updated.

TMoMoreau reviewed (approved):
Looks good, lets merge and test the links once the changes go live.
